constants/links.ts

```diff
@@ -32,13 +33,16 @@ export const EXTERNAL_LINKS = {
 		DocsRoot: 'https://docs.kwenta.io/',
 		FeeReclamation: 'https://docs.kwenta.io/resources/fee-reclamation',
 		HowToTrade: 'https://docs.kwenta.io/products/futures',
-		Governance: 'https://docs.kwenta.io/dao/governance-framework',
-		DaoRoles: 'https://docs.kwenta.io/dao/dao-roles',
+		Governance:
+			'https://app.radicle.xyz/seeds/maple.radicle.garden/rad:git:hnrkq1oex148yz4zi9tm7spfnjaryyuc93yay/tree/master/sections/2.md',
```

Comment:
Just want to double-check if `Governance` and `DaoRoles` are using the same link.

Reply:
Yes, correct, both point to the same section. Could update Governance to point to the general KSL. Will update :+1: